internal/pkg/flink/internal/utils/input.go

```diff
@@ -13,25 +13,32 @@ import (
 func GetStdin() *term.State {
 	state, err := term.GetState(int(os.Stdin.Fd()))
 	if err != nil {
-		log.CliLogger.Warnf("Couldn't get stdin state with term.GetState. Error: %v\n", err)
+		log.CliLogger.Warn("Couldn't get stdin state with term.GetState. Error: " + err.Error())
```

Comment:
Why aren't we using `Warnf()` anymore?

Reply:
Apparently [`fmt.Sprintf` is slower than direct concatenation of strings](https://forum.golangbridge.org/t/fmt-sprintf-vs-string-concatenation/23006). 
Since concatenating here doesn't take away from the readability of the string, we should just use `Warn`.

Reply:
I don't think performance matters here, especially since the difference between the two is on the order of nanoseconds. Let's just try to standardize the format throughout the code as much as possible.